src/main/java/com/dataloom/edm/internal/PropertyType.java

```diff
@@ -27,51 +28,71 @@ public PropertyType(
             @JsonProperty( SerializationConstants.TITLE_FIELD ) String title,
             @JsonProperty( SerializationConstants.DESCRIPTION_FIELD ) Optional<String> description,
             @JsonProperty( SerializationConstants.SCHEMAS ) Set<FullQualifiedName> schemas,
-            @JsonProperty( SerializationConstants.DATATYPE_FIELD ) EdmPrimitiveTypeKind datatype ) {
+            @JsonProperty( SerializationConstants.DATATYPE_FIELD ) EdmPrimitiveTypeKind datatype,
+            @JsonProperty( SerializationConstants.PII_FIELD ) boolean piiField ) {
```

Comment:
Why not `Optional<Boolean> piiField` and then `this.piiField = piiField.or( false )`?